docs/user/03-traces.md

```diff
@@ -288,17 +276,33 @@ metadata:
   name: backend
   namespace: default
 stringData:
-  endpoint: https://myhost:4317
+  endpoint: https://backend.example.com:4317
   user: myUser
   password: XXX
   token: Bearer YYY
 ```
 
-### Step 6: Rotate the Secret
+### Step 4: Rotate the Secret
 
 Telemetry Manager continuously watches the Secret referenced with the **secretKeyRef** construct. You can update the Secret’s values, and Telemetry Manager detects the changes and applies the new Secret to the setup.
 If you use a Secret owned by the [SAP BTP Service Operator](https://github.com/SAP/sap-btp-service-operator), you can configure an automated rotation using a `credentialsRotationPolicy` with a specific `rotationFrequency` and don’t have to intervene manually.
 
+### Result
+
+To activate and verify the constructed TracePipeline, follow these steps:
```

Comment:
same as above, why is activating the pipeline not part of the procedure? How common is it that users construct the pipeline but won't activate it?